### PR TITLE
feat: redirect user to profile after onboarding 

### PR DESCRIPTION
**Pull request by brandonroberts**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

- Redirects the user to their profile page after completing onboarding
- File formatting done in a separate commit

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Review by bdougie — APPROVED** [review]
Thanks for this quick fix.

File: pages/start.tsx [hunk 1]
```diff
@@ -113,7 +113,7 @@ interface LoginStep2Props {
 
 const LoginStep2: React.FC<LoginStep2Props> = ({
   handleLoginStep,
-  handleUpdateInterests: handleUpdateInterestsParent
+  handleUpdateInterests: handleUpdateInterestsParent,
```
Comment (OgDev-01) [inline pages/start.tsx]: i think we might need to turn off `trailingComma` in the prettier config

**Review by OgDev-01 — APPROVED** [review]
LGTM 🍕

File: pages/start.tsx (hunk 1 above)
Comment (brandonroberts) [reply pages/start.tsx]: I disabled the error in the eslint config. I think trailing commas are good for diffs. Open to suggestion though

**Pull request merged**